demos/human_pose_estimation_demo/python/human_pose_estimation_demo.py

```diff
@@ -96,12 +96,17 @@ def build_argparser():
 
 def get_model(ie, args, aspect_ratio):
     if args.architecture_type == 'ae':
-        Model = models.HpeAssociativeEmbedding
+        model = models.HpeAssociativeEmbedding(ie, args.model, target_size=args.tsize, aspect_ratio=aspect_ratio,
+                                               prob_threshold=args.prob_threshold)
+    elif args.architecture_type == 'hrnet':
```

Comment:
I see the reasons to have a special architecture type for this network, but the name IMO brings too much confusion. Which one is more "hrnet" HigherHRNet or EfficientHRNet (which is also "higher", BTW)?

Side note: don't you know if there is a way to save that kind of parameters in the IR itself, to make them closely attached to the network.

Reply:
I don't know this way. It might make sense to change name for type or is it better to use another method?

Reply:
IMO, I think that only storing postprocessing values (and any other values) inside the model file itself would save the day. Meanwhile, coming up with a better name seems to be a good thing.

Reply:
Names changed